webui/react/src/pages/Cluster/ClusterHistoricalUsage.tsx

```diff
@@ -92,8 +93,6 @@ const ClusterHistoricalUsage: React.FC = () => {
    * Load chart data.
    */
   useEffect(() => {
-    setChartSeries(undefined);
```

Comment:
i believe its fine to remove it. i dont know the reason why we need to set `undefined` first